leistungsbeschreibung/README.md

```diff
@@ -1,57 +1,57 @@
-
- 
+Um für verschiedene Plattformen und Betriebssysteme zur Verfügung zu stehen und um das Versprechen zu erfüllen, als einheitliche Schnittstelle zum Kassensystemen zu fungieren, ist die fiskaltrust.Middleware nach folgender Architektur aufgebaut.
 
   ![cashbox](../product-service-description/compliance-as-a-service/media/cashbox.png)
 
- 
-
-Der von der CashboxId identifizierte Konfigurationscontainer kann in verschiedene Plattformen und Betriebssysteme integriert werden. Die Verwaltung der Konfiguration und des Status dieser Komponenten erfolgt im marktbezogenen fiskaltrust-Portal. Der fiskaltrust-Sicherheitsmechanismus wird von der Queue-Komponente und der SCU-Komponente (Signaturerstellungseinheit) bereitgestellt, die die Bindung an die marktbezogenen Sicherheitsmechanismus-Anforderungen darstellen.
 
+Der durch die CashboxId identifizierte Konfigurationscontainer (Cashbox) kann in verschiedene Plattformen und Betriebssysteme integriert werden. Die Erstellung und Verwaltung der Konfiguration inklusive der dazugehörigen Komponenten erfolgt im marktbezogenen fiskaltrust.Portal. Der fiskaltrust.SecurityMechanism wird von der Queue-Komponente und der SCU-Komponente (Signaturerstellungseinheit) bereitgestellt. Die SCU-Komponente stellt die Anbindung an den marktbezogenen Sicherheitsmechanismus (z.B. TSE) dar.
 
 
 # Revisionssichere Daten-as-a-Service
 
-Kassenbetreiber erhalten revisionssichere Daten-as-a-Service, indem sie ein Pos-System verwenden, das die fiskaltrust.Middleware enthält. 
+Kassenbetreiber erhalten revisionssichere Daten-as-a-Service, indem sie ein Kassensystem verwenden, das die fiskaltrust.Middleware verwendet. 
 
-## fiskaltrust-cloud
+## fiskaltrust.Cloud
 
-Die fiskaltrust cloud befindet sich in einer tiefen Integration mit der fiskaltrust-Middleware, die die Daten als Teil des Sicherheitsmechanismus in die cloud spiegelt. Die Datenintegrität wird durch den Hash-Wert der Requests und Responses sichergestellt, und die Vollständigkeit kann durch Nachverfolgung der Kassenbeleg-Hash-Kette überprüft werden. Dies bedeutet, dass alle Originaldatensätze erhalten bleiben und unveränderlich sind. Der Schutz vor Datenverlust wird durch eine Lösung mit standortverteilten Daten-Speichern gewährleistet. Die Daten werden – entsprechend den steuerlichen Bestimmungen – in deutschen Rechenzentren gespeichert.
+Die fiskaltrust.Middleware, spiegelt die Daten als Teil des fiskaltrust.SecurityMechanism in die fiskaltrust.Cloud. Die Datenintegrität wird durch den Hash-Wert der Requests und Responses sichergestellt, und die Vollständigkeit kann durch Nachverfolgung der Hash-Kette überprüft werden (s.o.). Dies stellt sicher, dass alle Originaldatensätze erhalten bleiben und nicht verändert werden. Der Schutz vor Datenverlust wird durch unsere Lösung mit standortverteilten Daten-Speicherung gewährleistet. Die Daten werden in deutschen Rechenzentren gespeichert.
```

Comment:
"... Dies stellt sicher, dass ..." --> "Die Sicherung (Spiegelung) in die fiskaltrust.Cloud stellt sicher, dass"
"Der Schutz vor Datenverlust wird durch unsere Lösung ..." --> "Auch der Schutz vor Verlust der gesamten Daten wird durch unsere Lösung ..."
"Die Daten werden in deutschen Rechenzentren gespeichert." --> "Die Daten werden entsprechend der gesetzlichen Vorgaben in deutschen Rechenzentren gespeichert."

Reply:
Danke fürs Feedback, ich habe es in meinem aktuellen Commit berücksichtigt: https://github.com/christian-rogobete/productdescription-de-doc/commit/2805ee1b9398e0f0b6436d027e64f58da528cd86